app/scripts/modules/core/src/pipeline/config/triggers/trigger.html

```diff
@@ -1,106 +0,0 @@
-<div class="row">
-  <div class="col-md-12">
-    <div class="form-horizontal panel-pipeline-phase">
-      <div class="form-group row">
-        <div class="col-md-10">
-          <div class="row">
-            <label class="col-md-3 sm-label-right">Type</label>
-            <div class="col-md-9">
-              <select
-                class="input-sm"
-                ng-change="triggerCtrl.loadTrigger()"
-                ng-options="option.key as option.label for option in options"
-                ng-model="trigger.type"
-              >
-                <option value="">Select...</option>
-              </select>
-              {{description}}
-            </div>
-          </div>
-        </div>
-        <div class="col-md-2 text-right">
-          <button class="btn btn-sm btn-default" ng-click="triggerCtrl.removeTrigger(trigger)">
-            <span class="glyphicon glyphicon-trash" uib-tooltip="Remove trigger"></span>
-            <span class="visible-xl-inline">Remove trigger</span>
-          </button>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="col-md-10">
-          <div class="trigger-body"></div>
-        </div>
-      </div>
-      <div
```

Comment:
I'm not sure if this section got missed in the refactor, or if this behavior is captured by the `TriggerArtifactConstraintSelector` React Component

Reply:
There was some miscommunication on my part. I was under the impression we were no longer going to use the old expected artifacts any more and was going to just use the rewrite. I've added the old expected artifacts back into the code and added all the conditions back

Reply:
That may be true.  I don't personally know, because we don't (yet) use the Artifacts paradigm at Netflix.

Reply:
I added a discussion item for the status of the artifacts rewrite to our next UI SIG's agenda. Would be great if @jkschneider or someone else from the Pivotal team could give an update on the status of artifacts-rewrite documentation, scope of remaining work, and deprecation plan for pre-rewrite artifacts. I think the kubernetes provider is still in an inconsistent state where the deploy stage supports both artifacts paradigms but other stages (such as bake and patch manifest) only support the old paradigm.